packages/frontend/app/pods/components/pod/application/off-canvas-container/sidebar/contacts/component.ts

```diff
@@ -25,12 +25,13 @@ export default class ContactsSidebar extends Component<IArgs> {
   }
 
   get contacts() {
+    var sortedContacts: Contact[];
```

Comment:
Some notes on let vs const vs var 
https://medium.com/podiihq/javascript-variables-should-you-use-let-var-or-const-394f7645c88f

:)

that said, I should find a lint rule that discourages `var`

Reply:
oh actually, it's below, nevermind ;)